src/components/UI/AttachmentLink/AttachmentLink.tsx

```diff
@@ -37,10 +57,19 @@ export const AttachmentLinkUnStyled = ({ className, filename, data, type, path }
             <span className="filename">{filename}</span>
             {hasBase64 && <span className="filesize">({filesize})</span>}
           </p>
-          <p className="downloadtext mb-0">Download</p>
+          <div className="row no-gutters">
+            <a href={downloadHref} download={`${filename}`} className="downloadtext" data-testid="attachment-link">
+              Download
+            </a>
+            {redirectLink && (
+              <a href={redirectLink} target="_blank" rel="noopener noreferrer" className="downloadtext mb-0 ml-2">
+                Open
+              </a>
+            )}
+          </div>
```

Comment:
nitpicking here abit, to avoid this:

![Screenshot 2020-09-25 at 11 41 45 AM](https://user-images.githubusercontent.com/4774314/94224562-4ed9f080-ff25-11ea-8e32-e3dde0a0273c.png)

 in super tiny screens (edge case or what), maybe can do:


```suggestion
          <div className="row no-gutters">
            <div className="col-12 col-md-auto">
              <a href={downloadHref} download={`${filename}`} className="downloadtext" data-testid="attachment-link">
                Download
              </a>
            </div>
            {redirectLink && (
              <div className="col-12 col-md-auto ml-0 ml-md-2">
                <a href={redirectLink} target="_blank" rel="noopener noreferrer" className="downloadtext">
                  Open
                </a>
              </div>
            )}
          </div>
```

also, generally i think its mutually inclusive to use `.col` right after when u use `.row` (due to -15px margins on bootstrap `row` classes, then 15px padding on `col` classes)

it's something annoying to adhere to, but if u don't, think will have some hard time aligning stuff